kratos/processes/levelset_convection_process.h

```diff
@@ -774,6 +793,11 @@ class LevelSetConvectionProcess
         mConvectionElementType = GetConvectionElementName(element_type);
         std::string element_register_name = mConvectionElementType + std::to_string(TDim) + "D" + std::to_string(TDim + 1) + "N";
         mpConvectionFactoryElement = &KratosComponents<Element>::Get(element_register_name);
+        mElementRequiresLimiter =  ThisParameters["element_settings"].Has("include_anti_diffusivity_terms") ? ThisParameters["element_settings"]["include_anti_diffusivity_terms"].GetBool() : false;
```

Comment:
It is a good practice to keep the input json keys in the member variable names. Hence, I'd use `mIncludeAntiDiffusivityTerms`.

Reply:
You are right. Here, however, I think maybe calculate_gradient and calculate_limiter are more suitable because directly point to what we need (requirements).

Reply:
OK. Lets keep your last suggestion then. I think it is a good compromise between usability and code readability. 🆗